test/unit/grpc.component.unit.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback4-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+import {expect} from '@loopback/testlab';
+import {inject} from '@loopback/context';
+import {Application} from '@loopback/core';
+import {GrpcSequenceInterface, GrpcConfig, UnaryCall, UnaryReply} from '../../';
+import {grpc} from '../../src/decorators/grpc.decorator';
+import * as grpcModule from 'grpc';
+import {GrpcComponent, GrpcBindings} from '../..';
+import {
+  GreeterInterface,
+  HelloRequest,
+  HelloReply,
+} from '../protos/greeter.proto';
+
+const pkg: string = 'greeterpackage';
+const file: string = './test/protos/greeter.proto';
+
+describe('GrpcComponent', () => {
+  // GRPC Component Configurations
+  it('defines grpc component configurations', async () => {
+    const app: Application = givenApplication();
```

Comment:
Ditto - unit tests should not depend on other objects like `Application`. I think you should move this file to `test/acceptance/grpc.component.acceptance.ts`.

@virkt25 you have the most practical experience with testing a component. What's your recommendation - how to test the component itself?